nose/plugins/cover.py

```diff
@@ -121,6 +127,14 @@ def configure(self, options, conf):
             for pkgs in [tolist(x) for x in cover_packages]:
                 self.coverPackages.extend(pkgs)
         self.coverInclusive = options.cover_inclusive
+        self.coverInclusiveExcludes = []
+        if options.cover_exclude_file:
+            if isinstance(options.cover_exclude_file, (list, tuple)):
+                cover_exclude_file = options.cover_exclude_file
+            else:
+                cover_exclude_file = [options.cover_exclude_file]
```

Comment:
I suspect this if/else has to do with the default for the option.  I'm not sure I like the fact that it works only for a single file, but I think the answer is to provide a better default value than do this song and dance.